### DISCUSSION TITLE
Execution error with ReEDS v2022

### DISCUSSION THREAD

**Discussion by ssattler** [NREL/ReEDS-2.0#138] [Q&A: Execution] [answered]
I'm running the ERCOT section of the cases_test.csv file and I'm getting the following error, could you help me debug?

1 cases being run:
ref_seq

Only one case is to be run, therefore only one thread is needed
1

runbatch.py:613: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.
  os.path.join(inputs_case,'methane_leakage_rate.csv'))
runbatch.py:620: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.
  os.path.join(inputs_case,'ng_crf_penalty.csv')
runbatch.py:625: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.
  os.path.join(inputs_case,'co2_cap.csv')
runbatch.py:630: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.
  os.path.join(inputs_case,'co2_tax.csv')
runbatch.py:638: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.
  os.path.join(inputs_case,'gen_mandate_trajectory.csv')
runbatch.py:644: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.
  os.path.join(inputs_case,'gen_mandate_tech_list.csv')
runbatch.py:650: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.
  os.path.join(inputs_case,'climate_heuristics_yearfrac.csv')
runbatch.py:656: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.
  os.path.join(inputs_case,'climate_heuristics_finalyear.csv')
runbatch.py:662: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.
  os.path.join(inputs_case,'upgrade_costs_ccs_coal.csv')
runbatch.py:668: FutureWarning: The signature of `Series.to_csv` was aligned to that of `DataFrame.to_csv`, and argument 'header' will change its default value from False to True: please pass an explicit value to suppress this warning.
  os.path.join(inputs_case,'upgrade_costs_ccs_gas.csv')
test_1_18c_ref_seq has finished

The gamslog file is:
Starting calculation of financial parameters for C:\ReEDS_OpenAccess\runs\test_1_18c_ref_seq\inputs_case
df_ivt created for C:\ReEDS_OpenAccess\runs\test_1_18c_ref_seq\inputs_case
Traceback (most recent call last):
  File "C:\ReEDS_OpenAccess\input_processing\calc_financial_inputs.py", line 473, in <module>
    calc_financial_inputs(basedir, inputs_case)
  File "C:\ReEDS_OpenAccess\input_processing\calc_financial_inputs.py", line 140, in calc_financial_inputs
    columns=['t'])['finance_diff_real']
  File "C:\Anaconda3\lib\site-packages\pandas\core\frame.py", line 5628, in pivot
    return pivot(self, index=index, columns=columns, values=values)
  File "C:\Anaconda3\lib\site-packages\pandas\core\reshape\pivot.py", line 373, in pivot
    indexed = data.set_index(cols, append=append)
  File "C:\Anaconda3\lib\site-packages\pandas\core\frame.py", line 4184, in set_index
    index = ensure_index_from_sequences(arrays, names)
  File "C:\Anaconda3\lib\site-packages\pandas\core\indexes\base.py", line 5315, in ensure_index_from_sequences
    return MultiIndex.from_arrays(sequences, names=names)
  File "C:\Anaconda3\lib\site-packages\pandas\core\indexes\multi.py", line 336, in from_arrays
    raise ValueError('all arrays must be same length')
ValueError: all arrays must be same length

**Comment by wesleyjcole** [comment, marked as answer]
Did you change anything in the cases.csv file or in the cases_test.csv file?  If not, then are you using an up-to-date version of Python (my machine is running Python 3.9.7 as is running cases_test fine)?

**Comment by ssattler** [comment]
Thank you! I updated my version of Python from 3.7 to 3.9.7 and now cases_test scenario runs fine.
